sdk/src/lib.rs

```diff
@@ -66,9 +66,11 @@ pub mod entrypoint;
 pub mod entrypoint_deprecated;
 pub mod example_mocks;
 pub mod feature;
-pub mod genesis_config;
 #[cfg(feature = "full")]
-#[deprecated(since = "2.2.0", note = "Use `solana-hard-forks` crate instead")]
+#[deprecated(since = "2.2.0", note = "Use `solana_genesis_config` crate instead")]
+pub use solana_genesis_config as genesis_config;
+#[cfg(feature = "full")]
+#[deprecated(since = "2.2.0", note = "Use `solana_hard_forks` crate instead")]
```

Comment:
nit: kebab-case here too
```suggestion
#[deprecated(since = "2.2.0", note = "Use `solana-hard-forks` crate instead")]
```